src/services/form-validation.ts

```diff
@@ -81,6 +80,38 @@ validate.validators.attachment = function (
     return value.errorMessage
   }
 }
+function validateAttachments(
+  value: FormElementBinaryStorageValue[] | undefined,
+) {
+  const invalidAttachmentErrors = value?.reduce(
+    (invalidAttachments: string[], att) => {
+      const errorMessage = validateAttachment(att)
+      if (errorMessage) {
+        invalidAttachments.push(errorMessage)
+      }
+      return invalidAttachments
+    },
+    [],
+  )
+  if (invalidAttachmentErrors?.length) {
+    return `${invalidAttachmentErrors.join(', ')} could not be uploaded.`
```

Comment:
If the error message for 2/3 files failing is: 

> Failed to fetch

The validation message returned will be  

> Failed to fetch, Failed to fetch could not be uploaded.

Is this intended?

Reply:
No, I've just got the dumb